src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class RightTriangle extends Figure {
+    private static final String type = "triangle";
```

Comment:
The 'type' field should be an instance variable, not a static variable, as each figure might have a different type.